dotcom-rendering/src/web/components/Liveness.importable.tsx

```diff
@@ -144,6 +149,11 @@ export const Liveness = ({
 				insertNewBlocks(data.html);
 				hydrateBlocks();
 
+				if (lastUpdated) {
+					lastUpdated.setAttribute('dateTime', new Date().toString());
```

Comment:
Ideally this date would come from the new blocks being added, but thats not really included in the data we get from frontend. Setting the lastUpdated to Now instead _should_ hopefully be fine since we poll fairly often anyway.

Reply:
Hmm, that's actually a really good point. Should we display server time or client time?

I wonder what Frontend does here.

One point to consider is that the calculation by `timeAgo` is using `now` so perhaps always using local time is more accurate and less likely to result in strange results (here the reader's local time is wildly off)